docs/index-management/index-common-problems.md

```diff
@@ -0,0 +1,102 @@
+# Common Problems with Index Management
+
+## IllegalArgumentException when starting job
+
+When a reindexing job is started shortly after a the index has been
```

Comment:
These problems are specific to reindexing and index removal, but they now stand independent of that which is confusing in my opinion.
I'm not sure if it's really a good idea for that reason to split the current _Index Management_ page into three different pages as they are about the same topic, namely managing already existing indices.

Reply:
It currently only contains reindex and removal part. I see a lot of question to all of these sections.

The hole section is called Index Management. Index management should also contain creation.